From f1d3bc1f35db9c5811eb449a12a0cc31c0e88c6d Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Wed, 6 May 2026 09:57:59 +0800
Subject: [PATCH] configurations: nvidia: p3809-hmc: Add NvidiaMctpVdm config

Add NvidiaMctpVdm config to enable NSM management via nvidia-gpu sensor
service.

[1] https://gerrit.openbmc.org/c/openbmc/entity-manager/+/89948/1

Signed-off-by: Potin Lai <potin.lai@quantatw.com>
Change-Id: Ia61f0481f04b87ebf34d3d2d43fa6f36b26a757e
---
 configurations/nvidia/p3809-hmc.json | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configurations/nvidia/p3809-hmc.json b/configurations/nvidia/p3809-hmc.json
index a4f2133..b3ee330 100644
--- a/configurations/nvidia/p3809-hmc.json
+++ b/configurations/nvidia/p3809-hmc.json
@@ -886,6 +886,10 @@
             "Type": "SmbpbiVirtualEeprom",
             "Units": "Volts",
             "ValueType": "FLOAT64"
+        },
+        {
+            "Name": "NvidiaNSM",
+            "Type": "NvidiaMctpVdm"
         }
     ],
     "Name": "NVIDIA HMC",
-- 
2.52.0

